prompts/project_type/100_user_prompt.md

```diff
@@ -10,9 +10,7 @@ Here is a list of files that are currently versioned in this project:
 * {{.}}
 {{/project.files}}
 
-Use this list of files and the languages that we've detected in the project to 
```

Comment:
this seems like it might be a good suggestion but do you actually require this change for your use case?  I didn't see it being used.